assets/scss/shortcodes/tabbed-pane.scss

```diff
@@ -1,4 +1,4 @@
-.td-content {
+.tab-content {
   .highlight {
     margin: 0rem 0 2rem 0;
   }
```

Comment:
As I understand it, a code-block cannot appear in a `tab-content` outside of a `tab-pane`, but there is an override of `highlight` in `tab-pane` already, so I think that this style change is dead code and should be removed. WDYT @deining @LisaFC?

Reply:
@deining may have a better take on it - I'll take a look, but my immediate thought is possibly we might want a style for `highlight` content that's in a tabbed pane but not inside a code block. Though if they're the same style settings....

Reply:
Though hmm, we only apply `highlight` to code blocks, so that doesn't make sense.

Reply:
This fixes the style "leakage". Can you approve it as is @LisaFC?
We can address the possible style "dead code" separately.

Reply:
Sure

Reply:
We'll track this via:

- #1168